op-plasma/daserver.go

```diff
@@ -126,26 +130,52 @@ func (d *DAServer) HandlePut(w http.ResponseWriter, r *http.Request) {
 
 	input, err := io.ReadAll(r.Body)
 	if err != nil {
+		d.log.Error("Failed to read request body", "err", err)
 		w.WriteHeader(http.StatusBadRequest)
 		return
 	}
 
-	key := path.Base(r.URL.Path)
-	comm, err := hexutil.Decode(key)
-	if err != nil {
-		w.WriteHeader(http.StatusBadRequest)
-		return
-	}
+	if r.URL.Path == "/put" || r.URL.Path == "/put/" { // without commitment
+		// generate random commitment from random integer
```

Comment:
For this DA server, it should actually return the keccak or be disabled. This is a specific keccak-only DA server rather than generic DA server.